src/main/java/seedu/duke/parsers/Parser.java

```diff
@@ -14,8 +14,12 @@
  * Represents a Parser that parse a {@code Command}.
  */
 public abstract class Parser {
+    protected static final String EXIT_COMMAND_WORD = "exit";
+    protected static final String ADD_COMMAND_WORD = "add";
+    protected static final String LIST_COMMAND_WORD = "list";
+    protected static final String MARK_COMMAND_WORD = "mark";
+    protected static final String EMPTY_STRING = "";
 
```

Comment:
The Parser class is an abstract class, which means it does not serve for parse the command key words particularly, e.g. ("add", "exit"). Instead it serves as a util for all possible parser works. Hence I suggest to put the enums for command words in the ModHappyParser

Reply:
We could create a helper class to store all the constant strings

Reply:
#78